src/Gui/NaviCube.cpp

```diff
@@ -206,6 +203,9 @@ class NaviCubeImplementation {
     map<PickId, LabelTexture> m_LabelTextures;
 
     QMenu* m_Menu;
+
+    std::shared_ptr<NavigationAnimation> flatButtonAnimation;
+    SbRotation flatButtonTargetOrientation;
```

Comment:
This file uses the `m_` prefix notation for member variables: the new ones should match.